Plots/MapProjections/NCL_lcnative_1_lg.py

```diff
@@ -0,0 +1,109 @@
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/pre.8912.mon.nc"), decode_times=False)
+
+# Extract a slice of the data
+t = ds.pre.isel(time=0)
+
+###############################################################################
+# Plot:
+
+plt.figure(figsize=(14, 14))
+
+
+def Plot(row, col, pos, proj, title):
+    '''
+    Args:
+        
+        row (:class: 'int'): 
+            number of rows necessary for subplotting of visualizations
+        col (:class: 'int'): 
+            number of columns necessary for subplotting 
+        pos (:class: 'int'): 
+            position of visualization in m x n subplot
+        proj (:class: 'cartopy.crs'): 
+            which projection to visualize
+        title (:class: 'str'): 
+            center title of respective visualization
+    '''    
+    # Generate axes using Cartopy and draw coastlines
+    projection = proj
+    ax = plt.subplot(row, col, pos, projection=projection)
+    ax.set_extent((28, 57, 20, 47), crs=ccrs.PlateCarree())
+    ax.coastlines(linewidth=0.5)
+    # Import an NCL colormap
+    newcmp = gvcmaps.BlueYellowRed
+
+    
+    gl = ax.gridlines(draw_labels=True, dms=False, x_inline=False, y_inline=False)
+    gl.top_labels = True
+    gl.right_labels = True
+    gl.xlines = False
+    gl.ylines = False
+    gl.xlocator = mticker.FixedLocator([30,35,40,45,50,55])
+    gl.ylocator = mticker.FixedLocator([20,25,30,35,40,45])
+    gl.xlabel_style = {'rotation':0}
+    gl.ylabel_style = {'rotation':0}
+    
+    '''
+    When using certain types of projections in Cartopy, you may find that there
+    is not a direct 1-to-1 projection simularity. When looking at the three Lambert
+    projections offered, you will notice the closest match to the NCL projection
+    is actually the Lambert Cylindrical projection. This is due to NCL having certain
+    "smoothing" and "flattening" options for the Lambert Conformal projection not seen 
+    in the Cartopy version. By using Lambert Cylindrical over Lambert Conformal in Python,
+    you will be able to create the "rectangular" style of coordinates not classically 
+    represented by a Lambert Conformal map. Additionally, Cartopy does not currently support 
+    adding tick marks to a projection like NCL, this is why these Python projections do not 
+    have this feature. The GeoCAT Team is actively adding to the list of convenience functions
+    supported and hopes to add this functionality one day.
+    '''
+    
+    # Plot data and create colorbar
+    newcmp = gvcmaps.BlueYellowRed
+    t.plot.contourf(ax=ax, cmap=newcmp, transform=ccrs.PlateCarree(), 
+                    levels = 14, cbar_kwargs={"orientation":"horizontal", 
+                    "ticks":np.arange(0, 240, 20),  "label":'', "shrink":0.9})
+
+    plt.title(title, loc='center', y=1.17, size=15)
```

Comment:
Could these three lines be replaced by `gvutil.set_titles_and_labels`?

Reply:
I tried updating to using the gvutil version, but because we can't set the location of the titles/labels in either gvutil or with ax.gridlines, this will have to be hard coded unless one is updated to allow location settings